Tasks/Filippovich/task5-2.py

```diff
@@ -0,0 +1,9 @@
+
+def list(a, b, c, l=[]):
+    l.append(a)
+    l.append(b)
+    l.append(c)
+    l.reverse()
+    return l
+
+print(list("one","two","three"))
```

Comment:
the task was described like that:
"Write a function that takes a list of strings as an argument and returns a new list of strings that are all reversed."

your function should get a single argument which would be a list, go trough it and reverse all the strings inside it, something like that:
```
def reverse_items(l):
    res = []
    for i in l:
        if isinstance(i, str):
            res.append(i[::-1])
    return res
```
    
it takes a list of items, checks every item on being an str and appends a reversed item to a result